scripts/bootstrap.sh

```diff
@@ -304,13 +347,6 @@ setup()
 
     # fix OXA environment ownership
     chown -R $ADMIN_USER:$ADMIN_USER $OXA_PATH
-
-    # aggregate edx configuration with deployment environment expansion
-    # warning: beware of yaml variable dependencies due to order of aggregation
-    echo "---" > $OXA_PLAYBOOK_CONFIG
```

Comment:
(repeating) FYI: we moved this to happen as early as possible. we can now see what the oxa.yml is going to look like while we wait for the installation is running. this let's the user a) verify values are good ASAP and b) export credentials while they wait for the rest of the deployment to complete